2200031/10804.py

```diff
@@ -0,0 +1,18 @@
+T = int(input())
+
+for i in range(1, T+1):
+    string = input()
+    string_list = list(string)
+
+    for j in range(len(string_list)):
+        if 'b' in string_list[j]:
+            string_list[j] = 'd'
+        elif 'd' in string_list[j]:
+            string_list[j] = 'b'
+        elif 'p' in string_list[j]:
+            string_list[j] = 'q'
+        elif 'q' in string_list[j]:
+            string_list[j] = 'p'
```

Comment:
if문을 잘 나눠서 풀어주셨네요! 👍 
```suggestion
        if 'b' == string_list[j]:
            string_list[j] = 'd'
        elif 'd' == string_list[j]:
            string_list[j] = 'b'
        elif 'p' == string_list[j]:
            string_list[j] = 'q'
        elif 'q' == string_list[j]:
            string_list[j] = 'p'
```
지금은 두 문자열이 같은 지를 체크하는 것이기 때문에, `in` 연산자보다 `==` 를 활용해주시면 더욱 좋을 것 같습니다.